src/assessments/landmarks/test-steps/primary-content.tsx

```diff
@@ -10,19 +11,48 @@ import { AnalyzerConfigurationFactory } from '../../common/analyzer-configuratio
 import { ManualTestRecordYourResults } from '../../common/manual-test-record-your-results';
 import * as Markup from '../../markup';
 import { Requirement } from '../../types/requirement';
+import { autoPassIfNoLandmarks } from '../auto-pass-if-no-landmarks';
 import { LandmarkTestStep } from './test-steps';
 
 const description: JSX.Element = (
-    <span>The main landmark must contain all of the page's primary content.</span>
+    <span>
+        The <Markup.CodeTerm>main</Markup.CodeTerm> landmark must contain all of the page's primary
+        content.
+    </span>
 );
 
 const howToTest: JSX.Element = (
     <div>
-        <p>The visual helper for this requirement highlights the page's main landmark.</p>
+        <p>
+            The visual helper for this requirement highlights the page's{' '}
+            <Markup.CodeTerm>main</Markup.CodeTerm> landmark.
+        </p>
+        <p>
+            <Markup.Emphasis>
+                Note: If no landmarks are found, this requirement will automatically be marked as
+                pass.
+            </Markup.Emphasis>
+        </p>
         <ol>
             <li>
-                In the target page, examine the <Markup.CodeTerm>main</Markup.CodeTerm> landmark to
-                verify that it contains all of the page's primary content.
+                <p>Examine the target page to verify that all of the following are true:</p>
+                <ol>
+                    <li>
+                        The page has exactly one <Markup.CodeTerm>main</Markup.CodeTerm> landmark,
+                        and
+                    </li>
+                    <li>
+                        The <Markup.CodeTerm>main</Markup.CodeTerm> landmark contains all of the
+                        page's primary content.
+                    </li>
+                </ol>
+                <p>
+                    Exception: If a page has nested <Markup.CodeTerm>document</Markup.CodeTerm> or{' '}
```

Comment:
I considered suggesting we share some of the markup between primary-content and no-repeating-content because a good amount of the text is identical, but decided against it - it doesn't buy us much